lib/Module/Optimize.cpp

```diff
@@ -80,7 +84,11 @@ static cl::alias A1("S", cl::desc("Alias for --strip-debug"),
 
 // A utility function that adds a pass to the pass manager but will also add
 // a verifier pass after if we're supposed to verify.
+#if LLVM_VERSION_CODE >= LLVM_VERSION(3, 7)
+static inline void addPass(legacy::PassManager &PM, Pass *P) {
```

Comment:
> I realise at this stage that the tests might pass however I would like at the very least to still compile KLEE against LLVM 3.7.

Unfortunately there was no LLVM 3.7 in the linux system used in Travis when I tried.